src/security-council-mgmt/interfaces/ISecurityCouncilManager.sol

```diff
@@ -14,18 +14,30 @@ struct Roles {
     address memberRotator;
 }
 
+// data for a security council to be managed
+struct SecurityCouncilData {
+    address securityCouncil;
```

Comment:
since this same address is stored in the action contract, the one here in the struct isn't actually used anywhere (other than being emitted in events). still feels like maybe we want to keep it for clarity / sanity?